src/components/side_panel/conditional_formatting/cf_editor/cf_editor.ts

```diff
@@ -225,7 +224,33 @@ export class ConditionalFormattingEditor extends Component<Props, SpreadsheetChi
   }
 
   get errorMessages(): string[] {
-    return this.state.errors.map((error) => CfTerms.Errors[error] || CfTerms.Errors.Unexpected);
+    return this.state.errors.map((error) => this.errorMessage(error));
+  }
+
+  get firstInvalidRangeString(): string | undefined {
+    const sheetId = this.sheetId;
+    return this.state.ranges.find((xc) => {
+      const range = this.env.model.getters.getRangeDataFromXc(sheetId, xc);
```

Comment:
```suggestion
      const range = this.env.model.getters.getRangeFromSheetXc(sheetId, xc);
```
don't use RangeData when you do'nt have to. They are a bit ugly, and only meant for commands. We *shouldn't* want to access range._sheetId, it's "private"